pkg/authconfigmap/arn.go

```diff
@@ -0,0 +1,35 @@
+package authconfigmap
+
+import "github.com/aws/aws-sdk-go/aws/arn"
```

Comment:
I didn't know this package exists, we should adopt it in all places where ARNs are used.

Reply:
So you want to remove this type from the PR and solve it later on? Or keep it in here now as-is and centralize it for the entire project in #1218?

Reply:
No, keep it here, refactor later 👍